pilot/pkg/model/config.go

```diff
@@ -825,47 +825,123 @@ func (store *istioConfigStore) HTTPAPISpecByDestination(instance *ServiceInstanc
 	return out
 }
 
+// matchWildcardService returns true if wildcard matches the destination service
```

Comment:
"...if destinationHost matches svc, with the following match rules"? or something. wording is a bit confusing here.

Reply:
done